src/main/ImGuiZepEditor.cpp

```diff
@@ -478,7 +490,7 @@ ZepEditor& ImGuiZepEditor::GetEditor() const
 	return *m_editor;
 }
 
-void ImGuiZepEditor::Render(const char* id, const ImVec2& displaySize)
+void ImGuiZepEditor::Render(const ImVec2& displaySize)
```

Comment:
I'm confused why this is removed, doesn't that limit functionality to never have any children?